### PR TITLE
Connect ProjectInfo component; Update interface

### PR DESCRIPTION
**Pull request by niksonax**

### CODE REVIEW

File: frontend/src/common/types/project/project.ts [hunk 1]
```diff
@@ -20,5 +20,5 @@ export interface ProjectPage extends Project {
   instagramUrl?: string;
   facebookUrl?: string;
   dribbleUrl?: string;
-  finishDate: Date;
+  finishDate: string;
```
Comment (yevhenii-moroziuk) [inline frontend/src/common/types/project/project.ts:23]: why string?

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: frontend/src/common/types/project/project.ts (hunk 1 above)
Comment (niksonax) [reply frontend/src/common/types/project/project.ts:23]: `finishDate` returns string. `Date` type was placed by error

Comment (niksonax) [reply frontend/src/common/types/project/project.ts:23]: > why string?

`finishDate` returns string. `Date` type was placed by mistake

Comment (yevhenii-moroziuk) [reply frontend/src/common/types/project/project.ts:23]: We should return Date

Comment (yevhenii-moroziuk) [reply frontend/src/common/types/project/project.ts:23]: @dolphin323 will fix

**Pull request merged**